<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scorer</class>
 <widget class="QMainWindow" name="Scorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Legs</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1" colspan="5">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Player1Tab">
       <attribute name="title">
        <string>Player 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QTableWidget" name="Player1DartsTable">
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>75</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Score</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Bounce Out</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Knock Out</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Foul</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Remove</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Player2Tab">
       <attribute name="title">
        <string>Player 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="Player2DartsTable">
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>75</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Score</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Bounce Out</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Knock Out</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Foul</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Remove</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="1" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="LegNumberLabel">
      <property name="text">
       <string>4/14</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Sets</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="6">
     <widget class="DartboardView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="SetNumberLabel">
      <property name="text">
       <string>2/4</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="commit_turn_button">
      <property name="text">
       <string>Commit Turn</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="EndMatchButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>End Match</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>215</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScoreboard_View">
    <property name="title">
     <string>Scoreboard View</string>
    </property>
    <addaction name="actionMatch_Averages_2"/>
    <addaction name="actionMatch_Score_Stats_2"/>
    <addaction name="actionMatch_Highest_Out_2"/>
    <addaction name="actionMatch_Doubles_Triples"/>
    <addaction name="separator"/>
    <addaction name="actionPlayer_Ranks"/>
    <addaction name="actionPlayer_Last_Win"/>
    <addaction name="actionPlayer_Averages"/>
    <addaction name="actionPlayer_Score_Stats"/>
   </widget>
   <addaction name="menuScoreboard_View"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMatch_Averages">
   <property name="text">
    <string>Match Averages</string>
   </property>
  </action>
  <action name="actionMatch_Score_Stats">
   <property name="text">
    <string>Match Score Stats</string>
   </property>
  </action>
  <action name="actionMatch_Highest_Out">
   <property name="text">
    <string>Match Highest Out</string>
   </property>
  </action>
  <action name="action_Match_Statistics">
   <property name="text">
    <string>-- Match Statistics --</string>
   </property>
  </action>
  <action name="actionMatch_Averages_2">
   <property name="text">
    <string>Match Averages</string>
   </property>
  </action>
  <action name="actionMatch_Score_Stats_2">
   <property name="text">
    <string>Match Score Stats</string>
   </property>
  </action>
  <action name="actionMatch_Highest_Out_2">
   <property name="text">
    <string>Match Highest Out</string>
   </property>
  </action>
  <action name="actionMatch_Doubles_Triples">
   <property name="text">
    <string>Match Doubles/Triples</string>
   </property>
  </action>
  <action name="actionPlayer_Ranks">
   <property name="text">
    <string>Player Ranks</string>
   </property>
  </action>
  <action name="actionPlayer_Last_Win">
   <property name="text">
    <string>Player Last Win</string>
   </property>
  </action>
  <action name="actionPlayer_Averages">
   <property name="text">
    <string>Player Averages</string>
   </property>
  </action>
  <action name="actionPlayer_Score_Stats">
   <property name="text">
    <string>Player Score Stats</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DartboardView</class>
   <extends>QGraphicsView</extends>
   <header>DartboardView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
